conf/Makefile.chibios

```diff
@@ -39,7 +60,6 @@ Q=@
 RTOS_DEBUG ?= 0
 
 ifeq (,$(findstring $(RTOS_DEBUG),0 FALSE))
-$(info DEBUGGING ON)
```

Comment:
Why did you remove this info. I find it useful. Is it possible to at least keep the `DEBUGGING ON` ?